packages/components/resultCard.tsx

```diff
@@ -0,0 +1,33 @@
+import { Flex, Box, Avatar, Text } from '@chakra-ui/react';
+
+import type { NextPage } from 'next';
+
+const ResultCard: NextPage = () => {
```

Comment:
You should already parametrize this component as we will need. Remember you're implementing on a plataform your components on a PR should be final for others to use